include/verilated_vpi.cpp

```diff
@@ -24,12 +24,14 @@
 ///
 //=========================================================================
 
-#define VERILATOR_VERILATED_VPI_CPP_
+#include "vltstd/vpi_user.h"
 
-#include "verilated_vpi.h"
+#include <cstdio>
```

Comment:
I think these should be after verilated.h - generally we want verilatedos.h to be very first to work around portability problems.